.ci/smoke-tests.groovy

```diff
@@ -50,7 +50,29 @@ pipeline {
         dir ("${BASE_DIR}") {
           withGoEnv() {
             withTestClusterEnv {
-              sh(label: 'Run smoke tests', script: 'make smoketest')
+              script {
+                def smokeTests = sh(returnStdout: true, script: 'find ./testing/smoke -mindepth 1 -maxdepth 1 -type d').trim().split('\r?\n')
+                def smokeTestJobs = [:]
+                for (smokeTest in smokeTests) {
+                  smokeTestJobs["Run smoke tests in ${smokeTest}"] = {
+                    stage("Run smoke tests in ${smokeTest}") {
+                      sh(label: 'Run smoke tests', script: "make smoketest/run TEST_DIR=${smokeTest}")
+                    }
```

Comment:
This will run in parallel in the current CI worker, is that intended?

Reply:
This is intended as we are limited by the ESS deployment and as we run limited assertion in smoke tests.

Reply:
Will it cause any concurrency issue if they run in parallel in the same CI worker?

Reply:
As discussed with @marclop, all smoke tests are expected to be independent. If that's the case, then they can run on the same CI worker.